src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/email/TaskEmail.java

```diff
@@ -0,0 +1,34 @@
+package org.fogbowcloud.saps.engine.core.dispatcher.email;
+
+import com.google.gson.annotations.SerializedName;
+import org.fogbowcloud.saps.engine.core.archiver.storage.AccessLink;
+
+import java.util.Date;
+import java.util.List;
+
+public class TaskEmail {
+
+    @SerializedName("id")
+    private final String id;
+
+    @SerializedName("image_region")
+    private final String imageRegion;
+
+    @SerializedName("image_collection_name")
+    private final String imageCollectionName;
+
+    @SerializedName("image_date")
+    private final Date imageDate;
+
+    @SerializedName("access_links")
+    private final List<AccessLink> accessLinks;
+
+    public TaskEmail(String id, String imageRegion, String imageCollectionName, Date imageDate, List<AccessLink> accesslinks) {
```

Comment:
Pass a Saps Image Object to avoid many arguments in constructor.

Reply:
Done

Reply:
the idea of adding a simple constructor wasn't to avoid the complex one? why not deleting the complex one?

Reply:
Well remembered, I created the simple one and forgot to remove the complex because it was being used in the test, I fixed it